docs/demography.md

```diff
@@ -1822,9 +1822,9 @@ change as we make the migration rate less asymmetric:
 def get_mean_coaltimes(m):
     demography.set_migration_rate(source="A", dest="B", rate=m)
     debug = demography.debug()
-    TAA = debug.mean_coalescence_time([2, 0])
-    TBB = debug.mean_coalescence_time([0, 2])
-    TAB = debug.mean_coalescence_time([1, 1])
+    TAA = debug.mean_coalescence_time({"A": 2})
+    TBB = debug.mean_coalescence_time({"A": 1})
```

Comment:
Why the change? To match what it was before we need 
```
     TBB = debug.mean_coalescence_time({"B": 2})
```

Reply:
Yep, sorry. Will fix.